pom.xml

```diff
@@ -286,7 +286,7 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
       <version>${slf4jVersion}</version>
-      <scope>compile</scope>
+      <scope>provided</scope>
       <!-- mark the API as optional so it is not packaged in the HPI but available during compile -->
       <optional>true</optional>
```

Comment:
In this case we could delete `optional` I think?

Reply:
I actually see some uses of `optional` with `provided`. Please see `spotbugs-annotations` and `jcip-annotations`. While it's not very useful, I don't want to break the pre-existing pattern.